packages/flutter_html_video/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   flutter:
     sdk: flutter
   html: '>=0.15.0 <1.0.0'
-  flutter_html: '>=3.0.0-alpha.3 <4.0.0'
+  flutter_html: ^3.0.0-alpha.6
```

Comment:
Whilst not a big problem at all, as they are dependencies on our own libs, this does make it slightly less free for our users to, for example, use a `flutter_html` (core) 3.1.0 with flutter_html_video 3.0. Therefor, when we publish minor releases such as 3.1.0 we need to also release updates to our sub-libraries. Again, not a big problem at all but if melos supports it we might want to keep the less restrictive `<4.0.0`.

Reply:
That's a good point. Unfortunately, I'm not sure that melos supports that, and I could see how that is a little more complex to implement on their end.

I'll look a little deeper into that though and open a ticket with melos if there isn't one already, to see if that's something they're considering.